docs/resources/aws_ecs_service.md

```diff
@@ -0,0 +1,73 @@
+---
+title: About the aws_ecs_service Resource
+platform: aws
+---
+
+# aws\_ecs\_service
+
+Use the `aws_ecs_service` InSpec audit resource to test properties of a single specific Athena Work Group.
+
+The AWS::ECS::Service resource creates an Amazon Elastic Container Service (Amazon ECS) service that runs and maintains the requested number of tasks and associated load balancers.
+
+## Syntax
+
+Ensure that a services exists.
+    describe aws_ecs_service(cluster: "default", services: ["arn:aws:ecs:us-east-1:012345678910:service/ecs-simple-service"]) do
+      it { should exist }
+    end
+
+## Parameters
+
+`cluster, services` _(required)_
+
+For additional information, see the [AWS documentation on ECS Service](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-service.html).
+
+## Properties
+
+| Property | Description|
+| --- | --- |
+| services | A list of services to describe. You may specify up to 10 services to describe in a single operation. |
```

Comment:
as per this document if I give more than 1 service names to the api it will still return only 1 service result because of [this](https://github.com/inspec/inspec-aws/pull/370/files#diff-d967cde62d047c7f9fd8fadc4e6cf7c04f07ff63d1dc2f0abb41fe6303fbfa44R23)

Reply:
so this document description is completely invalid!